ui/desktop/electron-app/src/services/cache-daemon-manager.js

```diff
@@ -117,9 +117,18 @@ class CacheDaemonManager {
     return generateErrorPromise(stderr);
   }
 
-  search(requestData) {
+  async search(requestData) {
+    const start = Date.now();
+    // Log request data, but clean up token from log
+    const { auth_token_id, token, ...logRequestData } = requestData;
+
     if (isWindows()) {
-      return searchCliCommand(requestData);
+      const result = await searchCliCommand(requestData);
+
+      const end = Date.now();
```

Comment:
Do the `end` should be outside of this conditional? Or is the log intended just for windows platform?

I understand we are taking the start time `start` regardless the platform, but just taking `end` and logging if is windows, then we are just loggin for windows, correct?